README.md

```diff
@@ -1,5 +1,13 @@
-# Fragments-Download
-This script downloads anyurl by fragmenting the file and increases speed drastically.
+# All purpose downloader 
+## Powered by CRUx - The computing and programming club 
 
-usage: python3 frag.py URL
+### BITS Pilani Hyderabad Campus
```

Comment:
But someone happening to come to the README file probably doesn't care about who made this and where but rather, what it does. If they like it, then they'd look at other things. Again, its your project so you could take any route, but that's how I've seen it done.